test/ck_tile/moe_sorting/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-# Currently ck_tile is only built on gfx9
-if(GPU_TARGETS MATCHES "gfx9")
+# Currently ck_tile is only built on gfx9x and gfx95
```

Comment:
maybe better to enumerate the current valid options without a wildcard? gfx90a, gfx942, gfx950